docs/content/en/open_source/installation/running-in-production.md

```diff
@@ -87,22 +87,3 @@ You can execute the following command to see the configuration:
 
 `docker compose exec celerybeat bash -c "celery -A dojo inspect stats"`
 and see what is in effect.
-
-### Asynchronous Import
```

Comment:
Would it make sense to leave a short section behind that informs users the feature has been removed in 2.47.0? There might be users searching for async import in the docs.

Reply:
done